github_jira_sync_app/main.py

```diff
@@ -211,7 +211,8 @@ async def bot(request: Request, payload: dict = Body(...)):
     jira = JIRA(jira_instance_url, basic_auth=(jira_username, jira_token))
     jira_task_desc_match = f"This issue was created from GitHub Issue {issue.html_url}"
     existing_issues = jira.search_issues(
-        rf'project={settings["jira_project_key"]} AND description ~"\"{jira_task_desc_match}\""',
+        rf'project=\"{settings["jira_project_key"]}\" AND ' +
```

Comment:
why do you need escaping here?
we already have r-strings here

Reply:
fixed